client/Lykke.Service.MarketMakerReports.Client/Models/InventorySnapshots/Periodicity.cs

```diff
@@ -0,0 +1,13 @@
+namespace Lykke.Service.MarketMakerReports.Client.Models.InventorySnapshots
+{
+    public enum Periodicity
```

Comment:
How do you like name
Period {
    Unspecified,
    Tick
    Hour
    Day
    Day2
}

Reply:
Not quite sure, especially with "Day2" – not obvious what it mean. At my opinion current names of OnePerHour, OnePerDay, OnePerTwoDays are more straightforward and unambiguous.